.github/workflows/deps_lint.yml

```diff
@@ -271,10 +271,13 @@ jobs:
         pip install -U setuptools
         pip install -e .
         pip install -r requirements.txt
+        pip install -r requirements-dev.txt
```

Comment:
Not crazy about having to install all the development requirements as well here. Conceptually. Could we just install `invoke` stand-alone? Or do we want to make sure we install the `invoke` version we supplied in the development requirements?

Reply:
But maybe it's fine and it doesn't matter...

Reply:
This is how I did it initially, but you end up having to pull all the other deps from the top of tasks.py too

Reply:
(unless we restructure that too)

Reply:
Ah. Well - no. Leave it :)